src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/TextEditingTargetNotebook.java

```diff
@@ -753,13 +878,50 @@ else if (data.getType() == RmdChunkOutputFinishedEvent.TYPE_INTERACTIVE &&
             // set dirty state if necessary
             setDirtyState();
          }
+
+         if (satelliteChunks_.contains(data.getChunkId()))
+         {
+            forwardEventToSatelliteChunk(
+               docUpdateSentinel_.getId(),
+               data.getChunkId(),
+               new ChunkSatelliteOutputFinishedEvent(
+                  ensureVisible,
+                  data.getScope()
+               )
+            );
+         }
       }
    }
 
+   @Override
+   public void onConsoleWriteOutput(ConsoleWriteOutputEvent event)
```

Comment:
@jmcphers also, if I remove this event, I'm no longer getting the "command + enter" chunk executions in the satellite window. Would you expect to see that type of console output also being broadcasted to all satellite windows? Taking a closer look now...